pkg/process/util/signal_nowindows.go

```diff
@@ -18,6 +18,7 @@ func HandleSignals(exit chan bool) {
 	for sig := range sigIn {
 		switch sig {
 		case syscall.SIGINT, syscall.SIGTERM:
+			signal.Stop(sigIn)
```

Comment:
While you're hacking on this, would you please update `signal.Notify` to pass in the list of signals that we care about here?

This will become more important when the agent is updated to use go 1.14 as it will begin sending a lot of spurious `SIGURG` signals during runtime preemption

Reply:
Sorry for the delay, as you suggest to pass the signal to the Notify function what do you think of removing the SIGCHLD and SIGPIPE as there are just ignored later in the switch/case block ?

Reply:
Sounds good, thank you.